package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "small-client-wiki",
+  "version": "1.0.0",
+  "description": "Read-Only React Federated Wiki Browser",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/WardCunningham/small-client-wiki.git"
```

Comment:
Ah, gotcha. I looked on that very page, but only in the section about the `"repository"` key. Learn something new every day!